### ISSUE TITLE
Format path loops

### ISSUE THREAD

**Issue by nicholasdelrio** [nicholasdelrio/visko#15]
Need to ensure that nobody registers a sequence of operators that forms a loop.  In the future, I can be smarter about the loops and allow them under certain circumstances (don't know these circumstances yet).  For now, I just want to check for a loop and if one exists warn the user or nix the path.

**Comment by paulopinheiro1234** [comment]
Are you saying that we cannot register both 'JPEG2GIF' and GIF2JPEG'? Not
good :-) This restriction is something you may enforce locally. I doubt
one can enforce it globally.

Cheers,
Paulo.

On 4/26/12 3:47 PM, "Nicholas Del Rio"
<reply+i-4313053-ea31807296aef5ae39d03eae718dd8903ce9d626-1434744@reply.git
hub.com> wrote:

**Comment by nicholasdelrio** [comment]
You are absolutely correct, not good!  I can set some checks in the code to make sure the pipeline generator doesn't do something silly like building a pipeline such as (JPEG2GIF -> GIF2JPEG -> JPEG2GIF ....)
